### DISCUSSION TITLE
Mini Map and Free Zoom Option

### DISCUSSION THREAD

**Discussion by SilentMRG** [diasurgical/DevilutionX#6304] [Ideas]
Hello...

Would it be possible to add a mini map like the one in Diablo II? Opening the big map and having to keep throwing it around the corners of the screen to be able to see the character is a bit boring. Yes, I know, the Diablo map works like that, however, we are talking about a source port. =)

I know that the "Z" key enlarges the image, however, the image is very enlarged (if you know what I mean), would it be possible to add free zoom? The mouse wheel would be used to adjust the zoom.

Thank you very much for your attention!

**Comment by AJenbo** [comment]
Yeah we are thinking about adding such a map.

Gradual zoom would probably require GPU rendering so it's a big change and not til after 1.7.0
